### PR TITLE
Adding error code descriptions and examples to Budget API endpoints & other minor changes

### PR DESCRIPTION
**Pull request by barodapride**

### CODE REVIEW

File: src/api-reference/budget/budget-header.md [hunk 1]
```diff
@@ -581,4 +581,34 @@ Name | Type | Format | Description
...
+
+**Response**
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+```
+```json
+{
+  "status" : false,
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-header.md]: Something is wrong with either the markup or the JSON here.

**Comment by charlieconcur** [comment]
We need to fix the JSON issues present and have a peer reviewer approve the changes.

File: src/api-reference/budget/budget-header.md (hunk 1 above)
Comment (barodapride) [reply src/api-reference/budget/budget-header.md]: This should be fixed now, there was a missing " way above in the document

File: src/api-reference/budget/getting-started.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Examples:
 HTTPie:
```
Comment (charlieconcur) [inline src/api-reference/budget/getting-started.md]: There should be no links for specific tooling. Please follow the example in the [Documentation Template](https://github.concur.com/developer/api/blob/master/documentation-template.md).

File: src/api-reference/budget/budget-header.md (hunk 1 above)
Comment (charlieconcur) [reply src/api-reference/budget/budget-header.md]: Excellent.

**Review by charlieconcur — CHANGES_REQUESTED** [review]

**Comment by charlieconcur** [comment]
@barodapride are you a SAP Concur employee or customer?

**Comment by barodapride** [comment]
@charlieconcur yes I work on the budget team. Somebody should review this pull request soon.

**Comment by jtbigwoo** [comment]
- [x] Proofread documentation changes for spelling and grammatical errors
- [x] Used Markdown code blocks for all applicable examples using code
- [x] Used relative links when linking to other documentation on Developer Center
[N/A] If creating new API reference pages, followed the guidelines stated in the wiki
[N/A] If moving pages, added page redirects so users are redirected to the new page

These changes are approved, except for the change requested by Charlie.

**Comment by barodapride** [comment]
Updated documentation to get rid of HTTPie examples - replaced with http markdown. Also moved the request parameter info to the top as per the example documentation.

**Comment by charlieconcur** [comment]
I'm going to be a stickler here: We need all documentation to adhere to the documentation template which you can find at https://github.com/concur/developer.concur.com/blob/receipt-updates-and-polish/src/document-template.markdown. I am happy to do a co-working session if that will help you bring things into alignment.

**Comment by barodapride** [comment]
@mmeenapriya

**Comment by barodapride** [comment]
@charlieconcur I need some clarification as to what is not up to par. I more or less followed the template but there are some parts that are a bit different. Is there some model documentation somewhere I can see for an example?

**Comment by charlieconcur** [comment]
The documentation needs to follow the template flow. We are in the process of updating all documentation to be consistent.

**Comment by barodapride** [comment]
Closing, changes were moved into another request

**Pull request closed without merging**